.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Continuous Integration
+on:
+  push:
+    branches: [master]
+  pull_request:
```

Comment:
I said the same thing in his BM and and chef-cookbook PRs. Great minds... :) But for some reason it doesn't pick up the PRs from other branches that way which is fine in the normal case but means he can't test this PR. Which doesn't seem right, but was the problem he ran into.
Since we don't make PRs onto non-master branches, it shouldn't make any functional difference so I let it go.

Reply:
If I force this to master CI won't run for this PR (because it's on `gh-workflow`) so I won't be able to test it.